src/components/PostsTrendingCard/PostsTrendingCard.js

```diff
@@ -0,0 +1,28 @@
+import { allImage } from "../../constants";
+import React from "react";
+import "./PostsTrendingCard.scss";
+
+const PostsTrendingCard = () => {
+  return (
+    <div className="postsTrendingCard">
+      <div>
+        <img
+          src={allImage.momo}
```

Comment:
You can check out this [PR](https://github.com/itsnporg/blog-itsnp-frontend/pull/10/files#diff-6b6fe6d5d64274a3f623c5e7b5240b68e93fda3cb6b8353e892c49af03d0ea68) to create components that load content based on props provided to it.

Reply:
Done